def foo() -> None:
    return

def foo() -> None:  # Error: a function named 'foo' already exists
    return
